exercises/bowling/.meta/src/reference/java/BowlingGame.java

```diff
@@ -23,12 +23,12 @@ int score() {
                 if (rolls.size() <= frameIndex + 2) {
                     throw new IllegalStateException("Score cannot be taken until the end of the game");
                 }
-
+                
```

Comment:
This looks perfect, if you could just look at removing the extra space on the indicated lines I'll merge this in 🙂